jobs/JGLOBAL_WAVE_POST_SBS

```diff
@@ -9,9 +9,16 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_GRID
-
-mkdir -p "${COM_WAVE_GRID}"
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+    COMIN_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
+    COMIN_WAVE_GRID:COM_WAVE_GRID_TMPL \
```

Comment:
WAVE_GRID should only be OUT for this job.
```suggestion
    COMIN_WAVE_GRID:COM_WAVE_GRID_TMPL \
```

Reply:
This still hasn't been removed (I just fixed the suggested correction).

Reply:
This is causing the failure now on RDHPCS Hercules.